lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -338,7 +330,6 @@ E2Lib.optable_inv = {
 	dlt = "$",
 	trg = "~",
 	imp = "->",
-	fea = "foreach",
```

Comment:
what happened to that?

Reply:
It was never necessary - `E2Lib.optable` isn't used for keywords, they're all hardcoded in `tokenizer.lua`. (Strictly speaking this isn't a Luacheck issue, but I thought it wouldn't hurt to sneak it in.)